src/store/auth/reducer.js

```diff
@@ -0,0 +1,14 @@
+import { LOGIN, LOGOUT } from './actionTypes';
+
+export default (state = {}, action) => {
+  switch (action.type) {
+    case LOGIN:
+      return {
```

Comment:
you are  breaking a convention of not mutating the state here. There is no taking care about other (at this moment not existing fields)